frontend/pages/admin/IntegrationsPage/cards/MdmSettings/components/AutomaticEnrollmentSection/AutomaticEnrollmentSection.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from "react";
+import { InjectedRouter } from "react-router";
+import PATHS from "router/paths";
+
+import { AppContext } from "context/app";
+
+import PremiumFeatureMessage from "components/PremiumFeatureMessage";
+import SettingsSection from "pages/admin/components/SettingsSection";
+
+import AppleAutomaticEnrollmentCard from "./AppleAutomaticEnrollmentCard";
+import WindowsAutomaticEnrollmentCard from "./WindowsAutomaticEnrollmentCard";
+
+const baseClass = "automatic-enrollment-section";
+
+interface IAutomaticEnrollmentSectionProps {
+  router: InjectedRouter;
+  isPremiumTier: boolean;
+}
+
+const AutomaticEnrollmentSection = ({
+  router,
+  isPremiumTier,
+}: IAutomaticEnrollmentSectionProps) => {
+  const { config } = useContext(AppContext);
+
+  const navigateToWindowsAutomaticEnrollment = () => {
+    router.push(PATHS.ADMIN_INTEGRATIONS_AUTOMATIC_ENROLLMENT_WINDOWS);
+  };
+
+  const navigateToAppleAutomaticEnrollment = () => {
+    router.push(PATHS.ADMIN_INTEGRATIONS_AUTOMATIC_ENROLLMENT_APPLE);
+  };
+
+  return (
+    <SettingsSection title="Automatic Enrollment" className={baseClass}>
+      {!isPremiumTier ? (
+        <PremiumFeatureMessage alignment="left" />
+      ) : (
+        <div className={`${baseClass}__content`}>
+          <AppleAutomaticEnrollmentCard
+            viewDetails={navigateToAppleAutomaticEnrollment}
+            isAppleMdmOn={!!config?.mdm.enabled_and_configured}
+            configured={!!config?.mdm.apple_bm_enabled_and_configured}
```

Comment:
FYI, these config values are being [deprecated](https://github.com/fleetdm/fleet/pull/21043/files#r1705977965). We probably don't need to make more work for ourselves by updating everything now.

Reply:
ah thank you. I'll keep it in mind to update these when the API is changed.